src/components/Quickstart/QuickstartPanel.astro

```diff
@@ -0,0 +1,167 @@
+---
+import { MarkdownHeading } from "astro"
+import { QuickstartsFrontmatter, productsInfo } from "~/content/config"
+import * as CONFIG from "../../config"
+import TableOfContents from "./TableOfContents"
+
+interface Props {
+  frontmatter: QuickstartsFrontmatter
+  headings: MarkdownHeading[]
+  contentId: string
+}
+const { frontmatter, headings, contentId } = Astro.props
+const currentPage = new URL(Astro.request.url).pathname
+const currentFile = `src/pages${currentPage.replace(/\/$/, "")}.mdx`
```

Comment:
This url is wrong, this lives inside the content collection. Its probably worth checking if other uses of GITHUB_EDIT_URL point to the wrong url

Reply:
Good catch. The other docs do point to the content folder as well.